src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -2206,11 +2206,13 @@ def test_webapp_vnetE2E(self, resource_group):
         ])
         self.cmd(
             'webapp vnet-integration remove -g {} -n {}'.format(resource_group, webapp_name))
+        time.sleep(65)
```

Comment:
I'm curious the reason that we sleep `65`.

Reply:
During the 'webapp vnet-integration remove' call, we enter 20s wait for hit geo and 45s wait on app service side in order for NCs to be deleted successfully on machines.
So add time.sleep(65) for all az webapp vnet-integration related commands.

Reply:
Then define a constant for `65` and add the comments you provided above for the constant.